docs/app/components/usage-link/usage-link.component.ts

```diff
@@ -24,12 +24,18 @@ export class UsageLinkComponent {
     onShown() {
         this.popoverElement = (<any>this.popover)._popover._componentRef.location.nativeElement;
 
-        this.renderer.setStyle(this.popoverElement, 'style', '300px');
-        this.renderer.setStyle(this.popoverElement, 'maxWidth', '300px');
+        this.renderer.setStyle(this.popoverElement, 'style', '350px');
+        this.renderer.setStyle(this.popoverElement, 'maxWidth', '350px');
         this.renderer.setStyle(this.popoverElement, 'borderRadius', '0');
         this.renderer.setStyle(this.popoverElement, 'zIndex', '1');
     }
 
+    toggle(event: MouseEvent, popover: PopoverDirective) {
```

Comment:
Instead of passing in popover just use this.popover as it is already available in the class. (Also remove it from the .html file where you are passing it in)